File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    var ans = [];
    for (let i = 0; i < numbers.length - 1; i++) {
        for (let j = i + 1; j < numbers.length; j++) {
            if (!ans.includes(numbers[i] + numbers[j]))
                ans.push(numbers[i] + numbers[j]);
        }
    }
    ans.sort((a, b) => a - b);
    return ans;
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    ans = set()  # 중복 값 제거
    for i in range(len(numbers) - 1):
        for j in range(i+1, len(numbers)):
            ans.add(numbers[i] + numbers[j])
    
    return sorted(ans)  # 오름차순 정렬